tests/unit/models/gpflow/test_models.py

```diff
@@ -1502,34 +1502,44 @@ def test_sparse_variational_optimize(batcher: DatasetTransformer, compile: bool)
     assert model.model.training_loss(data) < loss
 
 
-def test_sparse_variational_gaussian_process_trajectory_sampler_raises_multi_latent_gp() -> None:
-    data = mock_data()
-    model = SparseVariational(svgp_model(*data, num_latent_gps=2))
-
-    with pytest.raises(NotImplementedError):
-        model.trajectory_sampler()
-
-
 @random_seed
 @pytest.mark.parametrize("use_mean_function", [True, False])
 @pytest.mark.parametrize("noise_var", [1e-5, 1e-2])
 @pytest.mark.parametrize("whiten", [True, False])
+@pytest.mark.parametrize("kernel_type", ["single", "shared", "separate"])
 def test_sparse_variational_trajectory_sampler_has_correct_samples(
     use_mean_function: bool,
     noise_var: float,
     whiten: bool,
+    kernel_type: str,
 ) -> None:
     x = tf.constant(np.arange(5).reshape(-1, 1), dtype=gpflow.default_float())
     y = _3x_plus_gaussian_noise(x)
     if use_mean_function:
         mean = gpflow.mean_functions.Linear()
     else:
         mean = gpflow.mean_functions.Zero()
+
+    num_latent_gps = 1
```

Comment:
Maybe put this in a helper function (or even a kernel-generating fixture)?

Reply:
I have now replaced this with the (improved) helper function to create the SVGP model directly.